src/plugins/intel_cpu/tests/unit/snippets/precision_propagation_convert_test.cpp

```diff
@@ -0,0 +1,163 @@
+// Copyright (C) 2022 Intel Corporation
```

Comment:
Why do we have this file in plugin unit tests not in snippets tests? We verify snippets logic so let's put it into [snippets  tests](https://github.com/openvinotoolkit/openvino/tree/master/src/common/snippets/tests/include/pass).

Reply:
no comments, moved : ) thanks